Assets/Plugins/Support/Scripts/Audio/AudioManager.cs

```diff
@@ -0,0 +1,123 @@
+using Support;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using NaughtyAttributes;
+namespace Support { 
+    public class AudioManager : MonoSingleton<AudioManager>
+    {
+        //getting resources
+        [Foldout("Source")]
+        [SerializeField]
+        [ReorderableList]
+        private Sound[] sounds;
+
+        [Foldout("Source")]
+        [SerializeField]
+        [ReorderableList]
+
+        private Sound[] music;
+        [Foldout("Source")]
+        [SerializeField]
+        [ReorderableList]
+        private Sound[] dialogs;
+        //Dictionary - fetch resources from Arrays to simplisify choosing a sound
+        private Dictionary<string,Sound> _sounds = new Dictionary<string, Sound>();
```

Comment:
You may confuse dictionaries' names with serialized properties. Probably you can rename it to smth like this
`_soundsDict`